cli/tests/cli-reference@.md.snap

```diff
@@ -187,8 +187,9 @@ To get started, see the tutorial at https://jj-vcs.github.io/jj/latest/tutorial/
 
    It is possible to run mutating commands when loading the repo at an earlier operation. Doing that is equivalent to having run concurrent commands starting at the earlier operation. There's rarely a reason to do that, but it is possible.
 * `--debug` — Enable debug logging
-* `--color <WHEN>` — When to colorize output (always, never, debug, auto)
-* `--quiet` — Silence non-primary command output
+* `--color <WHEN>` — When to colorize output (always, never, debug, auto)uto`
+
+quiet` — Silence non-primary command output
```

Comment:
what happened here?

Reply:
No idea honestly, I thought I just did `cargo nextest` and `cargo insta review`. Should be fixed now.